p5js/YOLO/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018 ml5
+// 
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+YOLO
+Real time Object Detection
+=== */
+
+let video;
+let yolo;
+let objects = [];
+
+function setup() {
+  createCanvas(640, 410);
+  video = createCapture(VIDEO, onVideoLoaded);
+  video.hide();
+  select('#start').mousePressed(function() {
+    detect();
+  });
+}
+
+function draw() {
+  image(video, 0, 0, 640, 410);
+  objects.forEach(function(object) {
+    noStroke();
+    fill(0, 255, 0);
+    text(object.className, object.x*width, object.y*height - 5);
+    noFill();
+    strokeWeight(4);
+    stroke(0,255, 0);
+    rect(object.x*width, object.y*height, object.w*width, object.h*height);
+  });
+}
+
+function onVideoLoaded() {
+  yolo = new ml5.YOLO(video.elt);
```

Comment:
A reminder to maybe revisit the question of if we can say things like `yolo = new ml5.YOLO(video)` for integration with p5?  Or is that opening too big of a can of worms? Sorry to be obsessed with this small detail :smile:

https://github.com/ml5js/ml5-library/issues/27

Reply:
Yes, totally! Just need to update the library.